v23.1/cockroachdb-feature-availability.md

```diff
@@ -137,12 +159,28 @@ $ cockroach sql --user=jpointsman --insecure
   Enter password:
 ~~~
 
-## Changefeed metrics labels
+### Core implementation of changefeeds
+
+The [`EXPERIMENTAL CHANGEFEED FOR`](changefeed-for.html) statement creates a new core changefeed, which streams row-level changes to the client indefinitely until the underlying connection is closed or the changefeed is canceled. A core changefeed can watch one table or multiple tables in a comma-separated list.
+
+### Changefeed metrics labels
 
 {% include {{ page.version.version }}/cdc/metrics-labels.md %}
```

Comment:
Feel free to save for a follow-up (that maybe Kathryn can take) as this was already here, but FWIW, this include provides a lot of content here. May be worth replacing it with the first paragraph from the description, and the key 'not available in cloud' bullet.

```suggestion
To measure metrics per changefeed, define a "metrics label" to which one or multiple changefeed(s) will increment each [changefeed metric](monitor-and-debug-changefeeds.html#metrics). Metrics label information is sent with time-series metrics to `http://{host}:{http-port}/_status/vars`, viewable via the [Prometheus endpoint](monitoring-and-alerting.html#prometheus-endpoint). An aggregated metric of all changefeeds is also measured.

Metrics labels are **not** available in {{ site.data.products.db }}```

Regardless that first sentence is hard to parse - FYI @kathancox.

Reply:
Will open a new issue

Reply:
Thanks! (Saw the issue.)